less/forum/Post.less

```diff
@@ -147,6 +147,7 @@
       border-radius: 0;
       display: block;
       overflow-x: auto;
+      max-height: 50vh;
```

Comment:
I think adding a minimum width might be a good idea, just in case. Maybe 300px?

Reply:
This would waste space for e.g. single line code block which are not uncommon. E.g.: https://forum.archlinux.de/d/34082-networkmanager-failed-nach-update

Reply:
Ah yes, my bad, what I was thinking of was using:

```css
max-height: 50vh;
max-height: max(50vh, 300px);
```

That way very small devices don't get tiiiiny code blocks, and big devices are still limited to 50vh. What do you think?

Reply:
Sure, this would stop the code block from getting too small. I guess you would need to switch to landscape mode on a small device to trigger this issue though.